docs/src/Tag.doc.js

```diff
@@ -0,0 +1,123 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTable from './components/PropTable.js';
+import Example from './components/Example.js';
+import PageHeader from './components/PageHeader.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Tag"
+    description="Tag is a object that holds text. It also has an x icon to remove it. The tag can appear within a form field or outside of it."
+  />
+);
+
+card(
+  <PropTable
+    props={[
+      {
+        name: 'disabled',
+        type: 'boolean',
+        defaultValue: false,
+        description:
+          'Set a disabled state so the tag looks inactive and cannot be interacted with.',
+      },
+      {
+        name: 'errorMessage',
+        type: 'string',
+        description:
+          "Set an error state on the tag. The message is used as an accessibility label for the error icon. Keep it short so it doesn't overwhelm the user.",
+      },
+      {
+        name: 'onRemove',
+        type: '({| event: SyntheticMouseEvent<> |}) => void',
+        description:
+          'Callback fired when the tag is removed. Should handle state updates to stop rendering the component. Required unless the tag is in a disabled state.',
+      },
+      {
+        name: 'removeIconAccessibilityLabel',
+        type: 'string',
+        description:
+          'Accessibility label for the icon button to remove the tag. Required unless the tag is in a disabled state.',
+      },
```

Comment:
Could we update to something like "Accessibility label for the icon button to remove the tag, ideally something like "Remove [Tag Name] Tag". Required unless the tag is in a disabled state. "

Could you also update the doc examples to be more specific, like "Remove New Tag" instead of just "remove"?